hw/ip/aes/dv/aes_model_dpi/aes_model_dpi.core

```diff
@@ -6,6 +6,9 @@ name: "lowrisc:dv:aes_model_dpi:0.5"
 description: "AES model DPI"
 filesets:
   files_dv:
+    depend:
+      - lowrisc:model:aes:0.5
+
     files:
       # Note: VCS needs the c/h-files to be treated as systemVerilogSource
```

Comment:
that looks pretty wrong. Is there a bug open somewhere?

Reply:
I don't think so. But @sriyerg pointed this out here: https://github.com/lowRISC/opentitan/pull/492#discussion_r335663388

Reply:
OK, so then I'm guessing it's only the VCS edalize backend which ignores non-RTL files because no handler is defined. Please open an issue and add the issue number to the comment, and I'll have a look.

Reply:
I have created the issue: #953

Reply:
Apologies, I had been meaning to file a bug for this but never got to it. Thanks for filing it. I hope it gets fixed soon.